### DISCUSSION TITLE
Channels with 0 users (sticky channels) should show first in QT client

### DISCUSSION THREAD

**Discussion by computertech1012** [BearWare/TeamTalk5#2430] [Ideas]
Hi,
The android client currently shows the channels with 0 users, also known as sticky channels, first in the channel list. I think that it would be good to also make this possible in the QT client, so that announcements on a server can be made and show up first if the channel has 0 users.
Maybe even add this for iOS client as well.
@bear101 What do you think of this idea? I think it's good to do it just like the android client has it.
Thanks for reading!

**Comment by CoBC** [comment]
You can already sort channels by popularity, it's not what you want?
